app/src/main/java/com/github/se/cyrcle/ui/profile/CreateProfileScreen.kt

```diff
@@ -28,29 +29,37 @@ fun CreateProfileScreen(navigationActions: NavigationActions, userViewModel: Use
   val defaultUser = User(UserPublic("", ""), UserDetails())
 
   val validationToastText = stringResource(R.string.create_profile_validation_toast)
+  val welcomeBonusText = stringResource(R.string.welcome_bonus_toast)
+  val combinedToastText = "$validationToastText\n$welcomeBonusText"
```

Comment:
what?

Reply:
There are two messages in one toast, saying that the user successfully created his profile and that he received a welcome bonus of 100 coins.

Reply:
Ah ok, I wasn't aware of this syntax